sdk/test-helpers/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@affinidi/test-helpers",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "scripts": {
+    "build": "npm run tsc",
+    "buildDocs": "typedoc --out docs src",
+    "start": "npm run build && node dist/index.js",
+    "tsc": "tsc -p tsconfig-build.json",
+    "lint": "eslint src/**",
+    "test": "",
+    "test:unit": "",
+    "test:integration": "",
```

Comment:
You could have the commands there (like in core-sdk etc), just not have any actual tests, to make adding them in the future easier.